static/sass/_patterns_navigation.scss

```diff
@@ -3,13 +3,24 @@
 @mixin maas-p-navigation {
   .p-navigation {
     background: $color-x-dark;
```

Comment:
You should set the new `$color-navigation-background` variable and then you can remove the color setting of the links.